lib/Form/RadioButton/Other.js

```diff
@@ -22,6 +22,29 @@ class Other extends Component {
   render() {
     const { name, value, id, checked, label, onChangeHandler, onTextChangeHandler } = this.props;
 
+    if (checked) {
+      return (
+        <div className="form__choice-element-wrapper">
+          <Input
+            name={name}
+            id={id}
+            checked={checked}
+            value={value}
+            onChangeHandler={onChangeHandler}
+          />
+          <input
+            className="form-radiobutton__other"
+            type="text"
+            id={`other-value-${id}`}
+            value={value}
+            onChange={onTextChangeHandler}
+            autoFocus
```

Comment:
Using this isn't a good idea for various reasons. One being that when loading the page this will auto focus for each other option checked.

What it is you wanted to achieve?